tests/test_scripts.py

```diff
@@ -49,6 +61,14 @@ def test_main_console(script_mod):
         script_mod.main_console()
 
 
+def test_train_bc_main(tmpdir):
```

Comment:
This fails because the data file is missing -- you need to set the path to `CARTPOLE_TEST_POLICY_PATH`.

Reply:
ah that's right. Fixed!